packages/common/src/view-model-maps/appeal-headline.js

```diff
@@ -54,3 +60,26 @@ exports.formatHeadlineData = (caseData, userType = APPEAL_USER_ROLES.INTERESTED_
 
 	return headlines;
 };
+
+/**
+ * @param {AppealCaseWithAppellant} caseData
+ * @param {AppealToUserRoles|LpaUserRole|null} userType
+ */
+const displayHeadlinesByUser = (caseData, userType) => {
+	const {
+		caseReceived,
+		appealValidDate,
+		lpaQuestionnaireSubmittedDate,
+		lpaQuestionnairePublishedDate
+	} = caseData;
+
+	if (userType === APPEAL_USER_ROLES.APPELLANT && caseReceived && lpaQuestionnairePublishedDate) {
+		return formatHeadlineData(caseData, userType);
+	} else if (userType === LPA_USER_ROLE && appealValidDate && lpaQuestionnaireSubmittedDate) {
+		return formatHeadlineData(caseData, userType);
+	} else {
+		return null;
+	}
+};
```

Comment:
Might be nicer to abstract all the conditions, especially since the return is the same in two of them

```
const shouldFormatHeadline = (
	{ caseReceived, appealValidDate, lpaQuestionnaireSubmittedDate, lpaQuestionnairePublishedDate },
	userType
) =>
	[APPEAL_USER_ROLES.APPELLANT, LPA_USER_ROLE].includes(userType) &&
	caseReceived &&
	appealValidDate &&
	lpaQuestionnairePublishedDate &&
	lpaQuestionnaireSubmittedDate;

const shouldNotFormatHeadline = (caseData, userType) => !shouldFormatHeadline(caseData, userType);

const displayHeadlinesByUser = (caseData, userType) => {
	if(shouldNotFormatHeadline(caseData, userType)) return null
	return formatHeadlineData(caseData, userType)
};
```

Reply:
Ive abstracted the logic out of the displayHeadlinesByUser but kept the if else as the 
`[APPEAL_USER_ROLES.APPELLANT, LPA_USER_ROLE].includes(userType) &&
	caseReceived &&
	appealValidDate &&
	lpaQuestionnairePublishedDate &&
	lpaQuestionnaireSubmittedDate;` 
does not differentiate between the 2 properties needed for each specific user